frontend/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -203,14 +205,15 @@ const SessionNumber = styled.p`
   font-size: 1.5vw;
   text-align: center;
   white-space: nowrap;
+  margin-bottom: 0;
 
   @media (max-width: 900px) {
     font-size: 3vw;
   }
 `;
 
 const SessionImage = styled.img`
-  max-width: 100%;
+  max-width: 60%;
```

Comment:
Something odd is going on here. If you resize your screen to just above 900 px, the session 1 image is much larger than the rest. It makes the vertical spacing odd for the top row. Could be a bug ticket though

<img width="405" alt="Screen Shot 2021-03-21 at 10 13 42 PM" src="https://user-images.githubusercontent.com/17476012/111931926-b3b87280-8a92-11eb-8538-1adfbff267e7.png">

Reply:
Hmmm this is interesting. I have **no** idea why it is happening, so I won't fix it in this pr. Do you want to make a bug ticket for it on Notion?

Reply:
No prob! Will put up a bug card later tonight